### ISSUE TITLE
Make sure neo4j dataitems are accessible from dataset

### ISSUE THREAD

**Issue by M-C-Mc** [TransparencyToolkit/Harvester#17]
The associated dataset is accessible from an individual dataitem, but the list of dataitems doesn't seem to be accessible from the dataset. Fix this and write here how to get the list/count.

**Comment by M-C-Mc** [comment]
This is not an issue in the show view. If you put binding.pry in the second line of the show action in the datasets controller, then put this in the terminal-
`@dataset.dataitems.each {|i| puts i}`
it should print a list of dataitem objects just fine. Similarly, `@dataset.dataitems.count` will get the number of data items for a dataset.

May still be an issue in the index view. Investigating further.

**Comment by M-C-Mc** [comment]
On the index view, neo4j throws this error message when attempting to access `dataset.dataitems` from within the iterator- Neo4j::Session::CypherError: Java heap space.

It doesn't throw an error from outside the iterator, when `@datasets.first.dataitems.each{ |i| puts i}` is run, but it also doesn't correctly print the list.

**Comment by M-C-Mc** [comment]
This seems to be fixed in the latest commit after I changed how associations work. Accessing `dataset.dataitems` in the dataset index view now works.

I also added some count helper methods- `get_count_per_dataset(dataset)`(to get the count of results for a single dataset) and `get_overall_count` (to get the count of all items). @bnvk - these don't need to be used on the frontend in this version, but these methods should give you the value needed to add result counts to the index page.
